.github/workflows/submitty_ci.yml

```diff
@@ -84,6 +87,7 @@ jobs:
 
 
   js-unit:
+    name: JavaScript unit tests
```

Comment:
To me adding 'test' or 'tests' to the end of the name feels kind of redundant, but that is debatable. 

Should we capitalize the first letter of each word? It seems like it would look a bit better to me, but again it's debatable.

Reply:
I suppose "test(s)" is a bit redundant but generally when I refer to these I say "unit test(s)" rather than just "unit," whereas I would say just "lint."

As for capitalization, I'm indifferent.